container/containerd.go

```diff
@@ -15,18 +15,35 @@ limitations under the License.
 package container
 
 import (
+	"bytes"
 	"context"
+	"encoding/json"
 	"fmt"
 	"io"
+	"os"
+	"strconv"
+	"strings"
+	"text/template"
+	"time"
 
 	"github.com/containerd/containerd"
-	//"github.com/containerd/containerd/namespaces"
+	"github.com/containerd/containerd/cio"
 	"github.com/containerd/containerd/namespaces"
 	refdocker "github.com/containerd/containerd/reference/docker"
+	gocni "github.com/containerd/go-cni"
+	"github.com/containerd/nerdctl/pkg/containerinspector"
+	"github.com/containerd/nerdctl/pkg/idutil/containerwalker"
```

Comment:
nerdctl is a docker compatible CLI for containerd , I highly doubt / not recommended from community side  to use the same just for common packages.

Reply:
> not recommended from community 

why?

> use the same just for common packages.

what could be an alternate then? copy the required funcs?